vagrant/config/init

```diff
@@ -0,0 +1,2 @@
+export KUBEADM_TOKEN=eskp57.iti9ygjwgvl3p8k2
```

Comment:
plase do not commit changes in the vagrant folder

Reply:
it is Ok I have added the vagrant folder to the .gitignore